.travis.yml

```diff
@@ -4,17 +4,8 @@ os:
   - osx
   - freebsd
 julia:
-  - 1.3
+  # - 1.0  # Latest LTS release (currently unsupported)
+  - 1    # Latest stable release
   - nightly
 notifications:
   email: false
-# script:
-#   - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
-#   - julia -e 'using Pkg; Pkg.clone(pwd()); Pkg.build("LibCURL"); Pkg.test("LibCURL"; coverage=true)'
-after_success:
-  - |
-    julia -e '
-      using Pkg
-      Pkg.add("Coverage")
-      using Coverage
-      Codecov.submit(process_folder())'
```

Comment:
Did you intend to remove coverage submission entirely?

Reply:
I'll restore that. I thought that the default Travis script now did this automatically.

Reply:
Nope, only if you do `codecov: true`